composer.json

```diff
@@ -16,18 +16,18 @@
         }
     ],
     "require": {
-        "php": ">=7.3.0,<7.5.0",
-        "doctrine/common": "^2.6",
-        "doctrine/dbal": "^2.5",
-        "doctrine/migrations": "^2.2",
+        "php": ">=7.4.0",
```

Comment:
Please lets constrain this, we don't support "every future version of PHP", and since PHP is not semver, it is safer to do:

```json
"php": "7.4.*|8.0.*",
```

Reply:
I rebased the PR, so this became irrelevant